### ISSUE TITLE
New icons

### ISSUE THREAD

**Issue by DragoCubed** [passionate-coder/Classic-Start#31]
thoughts?

**Comment by coddec** [comment]
Someone need to do the actual art work though...

**Comment by XenHat** [comment]
We're somewhat legally required to change the artwork. I'll pitch the idea to the artists I know.

**Comment by coddec** [comment]
> We're somewhat legally required to change the artwork. I'll pitch the idea to the artists I know.

Yes, upon Ivo's previous reply. 👍

**Comment by coddec** [comment]
God, you guys are all-rounders...
@XenHat  @ge0rdi  👍 👍 👍

**Comment by ibuprophen1** [comment]
@coddec, @XenHat & @ge0rdi... 

I'm pretty proficient with a number of image editors and such since I've been restoring Old Photos, Android App Icons/Artwork, etc... 

I may take a shot at this and provide you with some submissions to select from here. 

This challenge looks like I may only need some simple software like MS Paint, IrfanView and possibly Adobe PS. 

Hmmm... 😵


~Ibuprophen

**Comment by n9nu** [comment]
Count me in on anything that you guys could use help with. 

Tim S Dickerson
RHCA - Linux LCFS-LCFE - CompTIA Linux+
ARS: N9NU ARRL: VE/OES/AEC Aircraft ID: N944LA

**Comment by n9nu** [comment]
Can someone tell me what images they want changed? Uncle Sam let me take a two week vacation and I need to get back on my machines here and do some good. 

Thank you 'coddec' for taking this project over. I was going to, however, had to leave town for work and missed it. LOL. 

Tnx

Tim Dickerson
RHCA - Linux LCFS-LCFE - CompTIA Linux+
ARS: N9NU ARRL: VE/OES/AEC Aircraft ID: N944LA

**Comment by ibuprophen1** [comment]
@n9nu, this is regarding the software Icon itself...

You know the one that looks similar to the old Shell Gas Station Emblem? LOL! 

That brings back some memories... 😱 


~Ibuprophen

**Comment by ibuprophen1** [comment]
We just have to be careful and not use any "identical" Windows Images. 

Identical = The same exact look, color and (I'm serious) font. 

It would be okay (for example) If it included the old Windows 4 Colored flag with different colors and a slightly different appearance. 

I just wanted to keep from any type of Cease and Desist letters going out to anyone from Microsoft. 

Peace and Love! 
Peace and Love! 
~Ringo Starr 


~Ibuprophen

**Comment by XenHat** [comment]
We might want to put this issue on hold due to rebranding issues. I wouldn't want to have to ask the icons to be re-done twice. (#36 )

**Comment by n9nu** [comment]
Hi

Understood. I will stand by and see what evolves here. If ya need anything
else worked on, fire off an email and I will do what I can.

Tim




On Mon, July 9, 2018 15:22, Xenhat wrote:
> We might want to put this issue on hold due to rebranding issues. I
> wouldn't want to have to ask the icons to be re-done twice. (#36 )
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/passionate-coder/Classic-Start/issues/31#issuecomment-4
> 03607743


--------------------
Business Information
--------------------

Tim S Dickerson
FutureNET Systems
900 Vincent Drive
Suite C
Sandwich, IL 60548-2096

Service/Support: services@digital-audio-pro.us
Sales: sales@digital-audio-pro.us
Web-site: http://digital-audio-pro.us (Temp Page)

--------
Personal
--------

Email: n944la@gmail.com
Home: 815.312.2193 - VoIP
Cell: 815.508.8561
Amateur Radio Call-sign: N9NU
Sandwich Airport ICAO IS65) Unicom: 122.800
-------------------------
AIM: n9nu
kik: N944LA
Snapchat: N944LA
ICQ: 69881061
Skype: n944la - Messaging & VoIP
Facebook: Tim S Dickerson (dapstudios)
On-line Gaming ID: Reactor-Critical™ - Since 1998
--------------------------------------

**Comment by ibuprophen1** [comment]
@XenHat, this is very much understood but, I may get a few of them ready that could be a good "universal" selection that could be used to match a variety of names. 

Though, I'll lower the priority for this for now anyway. 

Thank you very much! 


~Ibuprophen

**Comment by yereverluvinunclebert** [comment]
No need to credit anyone for anything, just create artwork.

**Comment by Galaxy5151** [comment]
I've tried to make an icon for Classic Start.
It's not complete yet, but please tell me what you think of it.
![startorb2](https://user-images.githubusercontent.com/35960636/43391558-df0b9a8c-93e8-11e8-903c-1b3b9e73fbba.png)

**Comment by ibuprophen1** [comment]
How do you like these concept images @coddec, @XenHat and @ge0rdi? 

[Open-Shell-Concept-1.zip](https://github.com/NeoClassic-UI/Menu/files/2259153/Open-Shell-Concept-1.zip)

I kinda fell for this image and played a bit with different variations of it. 


~Ibuprophen

**Comment by XenHat** [comment]
I agree with perserving the "Shell" design, but I would prefer to avoid the classic colored frames from the old Windows Logo.

**Comment by ibuprophen1** [comment]
Huh? I saw to it that the color code wasn't the exact same but, those colors aren't Copywritten...

**Comment by Galaxy5151** [comment]
![startorb3](https://user-images.githubusercontent.com/35960636/43960715-c9464880-9caa-11e8-9bc0-eeb983f8d8b4.png)

**Comment by ibuprophen1** [comment]
@coddec, @XenHat & @ge0rdi...

How's the following unfinished one? 

![open-shell_concept-2-sm](https://user-images.githubusercontent.com/18060271/44010623-9a3ae978-9e81-11e8-9648-f5ee1f56716c.png)

I didn't add any text to it in hopes that you can let me know of where and what to state... 


~Ibuprophen

**Comment by Galaxy5151** [comment]
![startbutton](https://user-images.githubusercontent.com/35960636/44033815-6e274aa2-9f03-11e8-9652-e50f0388329f.png)
Made a start button version of my icon.

**Comment by Galaxy5151** [comment]
![startbutton](https://user-images.githubusercontent.com/35960636/44034364-d67ffaf8-9f04-11e8-8d21-2e8b7daee1fa.png)
Start button for if my icon is used.

**Comment by ibuprophen1** [comment]
@Galaxy5151, that does look very nice! 

Though this issue is regarding the Open Shell Logo image and not the start button icons themselves. 

Thank you very much for sharing this. I like simplistic ones! :-) 


~Ibuprophen

**Comment by jboutin** [comment]
> Like the actual seashell that is opened (just google images for "open shell").
Or is that crazy? 

It's not crazy. The original logo for ```classic shell``` was an image of a shell... which I always thought was misleading and awkward as it was a different use of the word than they used in the name of the product.

I never called Classic Shell 'CS', nor will I call Open Shell 'OS'. It's just Open Shell.

**Comment by ibuprophen1** [comment]
@coddec, @XenHat & @ge0rdi...

How's the following 2 New (unfinished) Concepts? 

This reflects that "Open Shell" look I believe that you were looking for... 
![open-shell_concept_3a-sm](https://user-images.githubusercontent.com/18060271/44116544-d3b9c1c0-9fde-11e8-885b-c4dd9728ee5e.png)

This reflects the same as the above with an added reflection style look... 
![open-shell_concept_3b-sm](https://user-images.githubusercontent.com/18060271/44116615-04541c9a-9fdf-11e8-95b0-b318f46133fa.png)

As before, they both have a transparent background. 

I really like the second one myself but, this is your call... 👍



~Ibuprophen

**Comment by ibuprophen1** [comment]
I apologize for the sizes of the images... LMAO! 

They're actually 1.5-2 inches... 

~Ibuprophen

**Comment by ibuprophen1** [comment]
@coddec, @XenHat & @ge0rdi...

How do you like the following? 

![open-shell_concept_3b-t1_sm](https://user-images.githubusercontent.com/18060271/44286538-6703bf80-a237-11e8-839e-0478f790963a.png)


~Ibuprophen

**Comment by AveYo** [comment]
That's.. too much psychedelic colors 💃 
The concept is solid ("an open shell") but it needs to be simple just like the original

**Comment by ibuprophen1** [comment]
Thank you very much for your suggestion! 👍

I know that the 2 of the 3 liked the direction i was going with original concept. 🙆

I also had inverted the classic windows colors as one of them suggested. 😁

I just need a little more input as to what/how they would like to go with this. 😱


~Ibuprophen 🍻

**Comment by XenHat** [comment]
I have to agree with Aveyo. I think the logo needs a more appealing feel, not try to exude how egdy we might be.
This is mostly subjective, but I prefer:
* clean, but soft edges
* smooth transitions of colors
* softer colors, closer to pastel.

I'm also affected by http://disorders.eyes.arizona.edu/category/clinical-features/blue-light-sensitivity. The current feel gives me a headache.

**Comment by XenHat** [comment]
>Do you want me to take my designs and shove it?
Not exactly. We appreciate all input and effort, and you've done a lot. But I think you should leave the art work to artists. I hate to say that and turn you down but the concepts aren't really up to par.

**Comment by ibuprophen1** [comment]
@XenHat, please don't think that I've got no experience in this field but, if your looking for a professional, that's okay with me and NO hard feelings my friend! :-) 


~Ibuprophen

**Comment by XenHat** [comment]
Wasn't invalidating your experience. I'm just not happy when the concept.

**Comment by Galaxy5151** [comment]
I know I'm a bit late, but here's the text-less version of my logo thingy.
![startorb3orig](https://user-images.githubusercontent.com/35960636/45982174-a0f28d80-c04f-11e8-8680-64bba4e006e2.png)

**Comment by XenHat** [comment]
I've been using that one for a while as my start meny icon, with the text. Could you post a menu version?

**Comment by Galaxy5151** [comment]
Sorry about being late again, but here it is:
![startbuttonv2](https://user-images.githubusercontent.com/35960636/47167668-e9236980-d2f6-11e8-8af5-34ff0a6c6303.png)

**Comment by david-rahrer** [comment]
Would there be any desire to go more flat as seems to be the trend now? The 3d button look seems a bit dated, though I'm no expert on creating graphics. I'm just noting the current state of GUI design. I'm glad there is an effort to modify the graphics for this new stage of the project. Thanks.

**Comment by bonzibudd** [comment]
I might be able to whip something up for this. I haven’t done graphics work in a while though, and Adobe keeps rearranging everything. Is this still a legal issue?

**Comment by bonzibudd** [comment]
I was thinking glyphs might be a good idea. I made a quick mockup for 16x16, just black-on-white. Not sure if it would work everywhere, but it would help look less tacky than the current design.
![image](https://user-images.githubusercontent.com/61938331/102001672-c6b4ed00-3cc2-11eb-910e-69b978501ec3.png)
![image](https://user-images.githubusercontent.com/61938331/102001683-daf8ea00-3cc2-11eb-806e-a8cf6b4cbb7f.png)

**Comment by bonzibudd** [comment]
I made a more functional icon from raster art. Here it is in case anyone is interested.
[OSM Glyph.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/5685805/OSM.Glyph.zip)

![image](https://user-images.githubusercontent.com/61938331/102028064-39cb6b80-3d76-11eb-918d-979aacfe431b.png)
![image](https://user-images.githubusercontent.com/61938331/102028173-be1dee80-3d76-11eb-8e7b-7568eefa1600.png)
![image](https://user-images.githubusercontent.com/61938331/102028031-0b4d9080-3d76-11eb-8a22-bdcab92aa1a3.png)

There would definitely need to be a colored icon as well. I think that a glyph icon would work best for the Classic Start button and sub-menus or context menus. Hopefully it's possible to account for dark/light backgrounds as well.

While I'm at it, here's a mockup with filled color:
![image](https://user-images.githubusercontent.com/61938331/102028645-3eddea00-3d79-11eb-9cb7-ed84a6c91607.png)

Additionally, if we're focusing on design in general, we could consider simplifying the installer background as well.

**Comment by bonzibudd** [comment]
I made a couple more refined icons just to try out. Here's a simple one with the classic color scheme:
![image](https://user-images.githubusercontent.com/61938331/120941575-5cb95580-c6f1-11eb-9ff7-a7e07d44202b.png)




Here's another one with a Windows 10/Fluent color scheme:
![image](https://user-images.githubusercontent.com/61938331/120941459-df8de080-c6f0-11eb-8748-2ac3039e27cb.png)

I'm personally leaning towards something like the bottom one, with the drop shadow and blue/gray border.

**Comment by bonzibudd** [comment]
Here's an alternative with more in-between colors:
![image](https://user-images.githubusercontent.com/61938331/120942602-07347700-c6f8-11eb-9cd3-d5034bf55621.png)
This one looks a bit better at smaller sizes.

**Comment by bonzibudd** [comment]
I tried another alternate one with the old colors and the drop shadow, but that combination doesn't look too good at size 32 and 16. I think the last one I posted will be the best in terms of color combination.

@XenHat @coddec any thoughts/suggestions?

**Comment by bonzibudd** [comment]
I made another one with a colored border, tried to make a balance between pastel and keeping a similar look to the original. I think this is one of the best so far. I also thickened the border slightly so that size 16/24 retain a sharper edge.
![image](https://user-images.githubusercontent.com/61938331/121733403-711a9b00-cac1-11eb-9588-35d3a9f06d1b.png)
In terms of the design, we would probably want to change the default orb as well, but I'm not too worried about that as it still has a similar look in terms of color/shape.

**Comment by XenHat** [comment]
This is nice

**Comment by bonzibudd** [comment]
I think I will end up using the blue/gray one instead of the orange border, as I was never able to get that one looking exactly how I wanted. I’ll start working on the installer artwork soon.

**Comment by bonzibudd** [comment]
Here's an updated version of the icon I posted earlier here: https://drive.google.com/file/d/1Q70YwDm03yDBN60I2EX6d9jKjFLoYeFA/view?usp=sharing
It's about the same as the earlier ones, but I had to restore it because I had saved the file incorrectly. It may or may not be the final design, as it's still a draft.

I think it's appropriate to consider updating the icon now, especially since there's a bigger distinction between Classic Shell and Open-Shell at this point.

**Comment by bonzibudd** [comment]
<img width="100" alt="image" src="https://user-images.githubusercontent.com/61938331/205552167-28123663-1bf9-4155-81a4-80a9ee85f973.png">
I think this will be the final color scheme, ignoring minor edits I may make in the future. After having some time to consider my older designs, I think this does a better job at balancing the saturation without being too tacky. I think my first few attempts felt a bit too dull/dark in context of the rest of Windows.

**Comment by ge0rdi** [comment]
@bonzibudd 
Just wanted to say that I like it :)

You can submit PR once you are satisfied with it.
I guess you want to replace `Src/Setup/OpenShell.ico` file. Because that one is the used in every component.

Also note that there is this _shell icon_ used in `Src/Setup/*.jpg` pictures that are used in installer screens.
Not sure how hard would be to replace it there too.

Also, have you considered to create aero button replacement in similar fashion?
I guess it would make sense to modernize it and make consistent with the new icon style as well.
Though no need to rush, one step at a time :)

**Comment by bonzibudd** [comment]
@ge0rdi
I’m glad you like it! Hopefully I’ll be done with it very soon.

For the installer art, I’ll probably make a slightly updated version of the whole screen for the sake of consistency. That shouldn’t take too long. 

I’ll definitely make an Aero button at some point, but that may take more time, so I’m not going to prioritize that yet.

**Comment by Gittyperson** [comment]
Thanks for the new artwork and brilliant 4.4.190 release! Just a curiosity: when opening the Open-Shell Menu settings, the standard Windows Explorer icon appears in the taskbar. Could that be replaced with an Open-Shell icon?

**Comment by ge0rdi** [comment]
> Could that be replaced with an Open-Shell icon?

That's good idea.
Explorer's icon is probably shown because the settings dialog comes from Explorer process.
I'll try to look into it.

**Comment by ge0rdi** [comment]
Hmm, I don't know how to change taskbar's icon for our settings dialog (it seems that taskbar always uses icon from main process).

Though it is possible to add _overlay_ to the icon. So it could look like this:
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2453094/4aa24e89-e516-4336-becf-ec3ead18e4a6)

I guess it is quite nice.

**Comment by ge0rdi** [comment]
It seems that there is a way to make separate taskbar button for `Open-Shell` settings dialog:
https://stackoverflow.com/a/24195594/1508446

This way there is new taskbar button added (not grouped with existing Explorer buttons) and it uses nice `Open-Shell` icon.
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2453094/7db2f6e1-45a7-4333-99e9-68aaa21e7ce3)

I will change it this way.

**Comment by Gittyperson** [comment]
Great, even better! Many thanks.

**Comment by ge0rdi** [comment]
You can check it out with build from #1636 PR.
